components/navigation/navigation.css

```diff
@@ -0,0 +1,61 @@
+.nav-boxx {
+    height: 5rem;
+    width: 80%;
+    /* border: 1px solid black; */
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    box-shadow: 0px 4px 12px -3px var(--text-color);
+    background-color: var(--primary-color);
+}
+
+.logo {
+    font-size: 2rem;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
```

Comment:
I think your code would work without this.

Reply:
I agree with Kartik here, as `justify-content` and `align-items` have a default value of `flex-start` I think you could avoid them here.